compositor_pipeline/src/pipeline/webrtc/whip_input.rs

```diff
@@ -39,6 +39,7 @@ impl WhipInput {
             return Err(InputInitError::WhipWhepServerNotRunning);
         };
 
+        let input_id = options.override_whip_session_id.unwrap_or(input_id);
```

Comment:
name it either url suffix or session id, but not input id, also check other places because you just swapped a value here, because some other calls might expect real input_id